src/imgui/Shortcuts.cc

```diff
@@ -0,0 +1,60 @@
+#include "Shortcuts.hh"
+#include "imgui_internal.h"
+
+#include "SettingsConfig.hh"
+
+#include <array>
+
+namespace openmsx {
+
+static constexpr auto defaultShortcuts = std::array{
+	Shortcuts::Data{ShortcutIndex::GOTO_MEMORY_ADDRESS, ImGuiMod_Ctrl | ImGuiKey_G, ShortcutType::LOCAL},
+	Shortcuts::Data{ShortcutIndex::GOTO_DISASM_ADDRESS, ImGuiMod_None, ShortcutType::LOCAL},
+};
+
```

Comment:
I know. At the time I was creating this prototype I did not understand the need for the `repeat` field yet.

Detail:
The original code had an `enum` instead of `bool` for `local`. That would make the `allShortcutInfo` table more readable. I mean now there are two boolean fields next to each other, it's not immediately clear which bool means what.